source/patterns/@aws-solutions-constructs/aws-events-rule-sns/README.md

```diff
@@ -0,0 +1,78 @@
+# aws-events-rule-sns module
+<!--BEGIN STABILITY BANNER-->
+
+---
+
+![Stability: Experimental](https://img.shields.io/badge/stability-Experimental-important.svg?style=for-the-badge)
+
+> All classes are under active development and subject to non-backward compatible changes or removal in any
+> future version. These are not subject to the [Semantic Versioning](https://semver.org/) model.
+> This means that while you may use them, you may need to update your source code when upgrading to a newer version of this package.
+
+---
+<!--END STABILITY BANNER-->
+
+| **Reference Documentation**:| <span style="font-weight: normal">https://docs.aws.amazon.com/solutions/latest/constructs/</span>|
+|:-------------|:-------------|
+<div style="height:8px"></div>
+
+| **Language**     | **Package**        |
+|:-------------|-----------------|
+|![Python Logo](https://docs.aws.amazon.com/cdk/api/latest/img/python32.png) Python|`aws_solutions_constructs.aws_events_rule_sns`|
+|![Typescript Logo](https://docs.aws.amazon.com/cdk/api/latest/img/typescript32.png) Typescript|`@aws-solutions-constructs/aws-events-rule-sns`|
+|![Java Logo](https://docs.aws.amazon.com/cdk/api/latest/img/java32.png) Java|`software.amazon.awsconstructs.services.eventsrulesns`|
+
+This AWS Solutions Construct implements an AWS Events rule and an AWS SNS Topic.
+
+Here is a minimal deployable pattern definition:
+
+``` javascript
+const { EventsRuleToSNSTopicProps, EventsRuleToSNSTopic } = require('@aws-solutions-constructs/aws-events-rule-sns');
+
+const props: EventsRuleToSNSTopicProps = {
+    eventRuleProps: {
+      schedule: events.Schedule.rate(Duration.minutes(5))
+    }
+};
+
+new EventsRuleToSNSTopic(stack, 'test-events-rule-sns', props);
```

Comment:
since stack is a variable and the module does not include a class which extends a construct this wont work here. Let me know if I am missing anything I followed the other integration test patterns?